### DISCUSSION TITLE
Sub-directory based on Tag that isn't always there

### DISCUSSION THREAD

**Discussion by jwynn6** [puddletag/puddletag#655] [Q&A]
I'm tagging audiobooks and want to place them in a directory scheme like %artist%/%series%/%album%/%title%. The issue arises when there isn't a series for a book. In attempting to use  %if(%series,%series%,) I end up with a directory named "".  It seems that there is a way to break this up in Actions to achieve it,  but I cannot wrap my head around the logic without having that %if.

**Comment by corubba** [comment]
Put the extra `/` into the if too.

`%artist%/$if(%series%,%series%/,)%album%/%title%`
